manager.py

```diff
@@ -125,7 +125,9 @@ def __init__(self,
             verbosity=self.v,
             logfile=self.logfile)
         # DEFAULT_UDP_PORT and DEFAULT_TCP_PORT are assigned in sender
-
+        self.branch = 'master'
+        self.tag = 0
```

Comment:
I don't think you need the `tag` attribute. If the number is 0, nothing happens. If the number is 1, you set `self.quit_after_interval` and that's it. But I like keeping `branch` as an attribute.